README.md

```diff
@@ -79,7 +57,25 @@ If you would like to develop against the hosted custom badgekit-api we have runn
         export ORCID_REDIRECT_URI=#############
 
 Ask [@acabunoc](http://github.com/acabunoc) for ones marked `###########`. Our custom BadgeKit API code can be found [here](https://github.com/acabunoc/badgekit-api).
+Feel free to change `PORT` to any available port.
 
+#### Run using Docker
+
+You can use Docker to bring up a quick instance of the app to develop against. This way you don't need to have node, MongoDB, and redis installed on your host.
+
+* Make sure you have [Docker](https://www.docker.com/) (>=1.10) and docker-compose (>=1.6) installed.
+* Setup your environment variables as explained in the previous section
+* start the service with `docker-compose up`
+* visit the running service
+  * If on Linux host: http://localhost:5000
+  * If not Linux: http://(docker_host_ip):5000 (You can find your docker IP with `docker-machine ip default`)
+
+
+#### Run locally
+
+* Install PaperBadger's Node dependencies: `npm install`
+* Make sure MongoDB and redis are running and locally accessible.
+* Setup your environment variables as explained in a previous section
 * Run `npm start`, and open up `http://localhost:5000/` in your favourite web browser!
 
 To run the application successfully you need to have [mongodb](https://www.mongodb.org/) server and [redis-server](http://redis.io/download) running locally. You can install these from their official website or use your favorite package manager.
```

Comment:
(err... 77.  The line that says `* Make sure MongoDB and redis are running and locally accessible.`)